tests/transaction.js

```diff
@@ -15,7 +15,7 @@ if (
     madWallet = new MadWalletJS(process.env.CHAIN_ID, process.env.RPC);
 }
 else {
-    let madWallet = new MadWalletJS();
+    madWallet = new MadWalletJS();
```

Comment:
This block of code is reassigning the variable unnecessarily but fixing it breaks a few unit tests.
![image](https://user-images.githubusercontent.com/5488554/152809358-40e93003-230e-44c0-98c4-29dda3106a79.png)

Reply:
These appear to be failing because an RPC isn't set As all of those functions rely on an RPC endpoint somewhere in the stack. If process.env isn't available and the fallback is on this line these tests will fail. 

What is interesting is that I am being led to believe you have process.env set and it is failing when you remove the let?